solutions/standard/variables.tf

```diff
@@ -16,22 +16,17 @@ variable "resource_tags" {
 
 variable "prefix" {
   type        = string
-  description = "Prefix to add to all resources created by this deployable architecture . To not use any prefix value, you can set this value to `null` or an empty string."
-  default     = "dev"
+  description = "Prefix to add to all resources created by this deployable architecture. To not use any prefix value, you can set this value to `null` or an empty string."
 }
 
-variable "use_existing_resource_group" {
-  type        = bool
-  description = "Whether to use an existing resource group."
-  default     = false
-}
-
-variable "resource_group_name" {
+variable "existing_resource_group_name" {
   type        = string
-  description = "The name of a new or an existing resource group to provision resources in to. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format. If not set a new resource group will be created using the prefix variable value."
+  description = "The name of the existing resource group to provision resources in to."
+  nullable    = false
+  default     = "Default"
 }
 
-variable "ease_name" {
+variable "easinstance_name" {
```

Comment:
see my comments in https://github.com/terraform-ibm-modules/terraform-ibm-enterprise-app-java/pull/63